apps/ledger-live-mobile/e2e/page/accounts/account.page.ts

```diff
@@ -1,43 +1,40 @@
+import { getElementById, getTextOfElement, scrollToId, tapByElement } from "../../helpers";
 import { expect } from "detox";
-import {
-  currencyParam,
-  openDeeplink,
-  waitForElementById,
-  getElementById,
-  tapByElement,
-} from "../../helpers";
-
-const baseLink = "account";
+import jestExpect from "expect";
 
 export default class AccountPage {
-  accountSettingsButton = () => getElementById("accounts-settings");
-  assetBalance = () => getElementById("asset-graph-balance");
-  accountTitleId = (assetName: string) => `accounts-title-${assetName}`;
-  accountAssetId = (assetName: string) => `account-assets-${assetName}`;
-
-  async waitForAccountPageToLoad(assetName: string) {
-    await waitForElementById(this.accountTitleId(assetName));
-  }
+  accountGraph = (accountId: string) => getElementById(`account-graph-${accountId}`);
+  accountBalance = (accountId: string) => getElementById(`account-balance-${accountId}`);
+  accountSettingsButton = () => getElementById("account-settings-button");
+  accountAdvancedLogRow = () => getElementById("account-advanced-log-row");
+  operationHistorySectionId = (accountId: string) => `operations-history-${accountId}`;
+  accountScreenScrollView = "account-screen-scrollView";
+  accountAdvancedLogsId = "account-advanced-logs";
 
-  async expectAccountBalanceVisible() {
-    await expect(this.assetBalance()).toBeVisible();
+  async openAccountSettings() {
```

Comment:
do you think we need to add @step above each method to have the same level of detail in Allure report?

Reply:
Sure, I will add them

Reply:
Done